kie-server-parent/kie-server-services/kie-server-services-openshift/src/test/java/org/kie/server/services/openshift/impl/storage/cloud/KieServerStateOpenShiftRepositoryRegularTest.java

```diff
@@ -394,5 +432,11 @@ public void testKieContainerUpdateDuringRolloutAllowed() {
         assertTrue(repo.isKieContainerUpdateDuringRolloutAllowed(cfmContainer, stateWithStopped));
         // Does not allow state transition from STOPPED to STARTED as it is not necesscary
         assertFalse(repo.isKieContainerUpdateDuringRolloutAllowed(cfmStopped, state));
-}
+    }
+
+    @Test
+    public void testLableSelector() {
+        LabelSelector selector = repo.getKieServerCMLabelSelector();
```

Comment:
Please adjust the test to check content of LabelSelector. Add also test checking functionality of global discovery switch.

Reply:
Will address this in the next commit.